components/script/dom/element.rs

```diff
@@ -2575,6 +2616,28 @@ impl Element {
         self.set_click_in_progress(false);
     }
 
+    pub fn get_lang(&self) -> Atom {
+        if let Some(ref lang) = *self.cached_lang.read() {
+            lang.clone()
```

Comment:
Yes, but the layout code doesn't call this function anymore, `fn language()` for `LayoutJS<Element>` now does its own attribute/parent node traversal.